llvm/lib/CodeGen/LivePhysRegs.cpp

```diff
@@ -262,7 +262,7 @@ void llvm::addLiveIns(MachineBasicBlock &MBB, const LivePhysRegs &LiveRegs) {
   const MachineRegisterInfo &MRI = MF.getRegInfo();
   const TargetRegisterInfo &TRI = *MRI.getTargetRegisterInfo();
   for (MCPhysReg Reg : LiveRegs) {
-    if (MRI.isReserved(Reg))
+    if (TRI.getReservedRegs(MF).test(Reg))
```

Comment:
Can you explain why this change is required?

Reply:
No longer required with simple MSR write. Removed.